test/clients.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2020 VMware, Inc.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package test
+
+import (
+	"time"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"knative.dev/pkg/test"
+
+	// Support running e2e on GKE
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
```

Comment:
do you want to always import this?

Reply:
It's test code, so I don't think it really affects/hurts application side, so yolo.